jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/DatePickersTests.java

```diff
@@ -115,7 +116,7 @@ public class DatePickersTests extends TestsInit {
     private static final List<String> CHINESE_MONTHS = Arrays.asList("1月", "2月", "3月", "4月",
             "5月", "6月", "7月", "8月", "9月", "10月", "11月", "12月");
 
-    @BeforeClass
+    @BeforeMethod
```

Comment:
Yes. In case we use @BeforeClass then failed test can bring some flakyness to other.
Besides that @BeforeMethod gives us insurance that anyone who changes tests will not make them dependant.
If someone uses @BeforeClass then temptation to create dependencies between tests can appear insensibly. With @BeforeMethod we guarantee that it will not unless someone changes annotation.